### ISSUE TITLE
babel-preset-env has no changelog

### ISSUE THREAD

**Issue by StephanBijzitter** [babel/babel#7927]
Dependabot is proposing an update for us, but it is unable to parse any information from the repository to provide in the pull request for us.

Of course, I then tried to find the recent changes myself, as I'd like to know what the new feature is (maybe it could be useful for us!), but I myself also cannot find the information I need.

As such, I'd like to ask what new feature has been introduced in `babel-preset-env` and if the missing changelog is a bug, report it as such.

<img width="771" alt="screen shot 2018-05-14 at 12 54 02" src="https://user-images.githubusercontent.com/1649903/39993451-2cfe24e4-5776-11e8-8ed0-2ab95b316a38.png">

**Comment by existentialism** [comment]
@StephanBijzitter thanks for the issue. the only change was bump to browserslist (https://github.com/babel/babel-preset-env/pull/458) so that newer browsers would be supported. It may be hard to follow since the changed was landed/published from the old repo, that is in archive mode.

**Comment by StephanBijzitter** [comment]
Ah, pretty cool! Sadly our users are still stuck on the older browsers, but now I at least know what I'm about to merge. Thanks!
